python/lsst/sims/GalSimInterface/galSimCatalogs.py

```diff
@@ -191,13 +192,93 @@ def get_sedFilepath(self):
         return numpy.array([self.specFileMap[k] if k in self.specFileMap else None
                          for k in self.column_by_name('sedFilename')])
 
+    def _calcGalSimSed(self, sedName, zz, iAv, iRv, gAv, gRv, norm):
```

Comment:
Can you make this name more significantly different from `_calculateGalSimSeds` to avoid confusion.  Maybe `_calculateSingleGalSimSed`?